package/utils/omnia-mcutool/Makefile

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright (C) 2016-2023 CZ.NIC z.s.p.o. (http://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=omnia-mcutool
+PKG_VERSION:=0.2
+PKG_RELEASE:=$(AUTORELEASE)
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
+PKG_SOURCE_URL:=https://gitlab.nic.cz/turris/omnia-mcutool/-/archive/$(PKG_VERSION)/
+PKG_HASH:=e1b823dac9f908b86ce63e7f525a70fa40c004bcfee1c4766cfcf52dfd58382d
+
+PKG_MAINTAINER:=Marek Mojik <marek.mojik@nic.cz>
+PKG_LICENSE:=GPL-2.0-or-later
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/omnia-mcutool
+  SECTION:=utils
+  CATEGORY:=Utilities
+  URL:=$(PKG_SOURCE_URL)
+  TITLE:=CZ.NIC Turris Omnia MCU utility
+  DEPENDS:=+omnia-mcu-firmware @TARGET_mvebu_cortexa9_DEVICE_cznic_turris-omnia
+endef
+
+define Package/omnia-mcutool/description
+The omnia-mcutool utility is mainly used to upgrade the firmware on the
+microcontroller on the Turris Omnia router. It can also show state of MCU
+settings and configure MCU options (GPIOs, LEDs, power).
```

Comment:
```suggestion
  The omnia-mcutool utility is mainly used to upgrade the firmware on the
  microcontroller on the Turris Omnia router. It can also show state of MCU
  settings and configure MCU options (GPIOs, LEDs, power).
```

Reply:
I disagree, because then in `make menuconfig` it looks like this
![menuconfig](https://github.com/openwrt/openwrt/assets/1883885/27c1e59f-1b57-4aae-884d-d0a3f2386e93)

Notice that the indentation does not take on the first line, only from the second one. It seems that it does not work correctly for multi-line descriptions.

Looking at the tree with `git grep -A 2 'Package/.*/desc', there are some packages with multi-line descriptions that do indent (like `kobs-ng`, `gperf`) and some that do not (`gdb`, `gdbserver`).